common-rendering/src/editorialPalette.ts

```diff
@@ -87,6 +92,44 @@ const textHeadlineDark = (format: ArticleFormat): Colour => {
 	}
 };
 
+const textKicker = (format: ArticleFormat): Colour => {
+	switch(format.theme) {
+		case ArticlePillar.News:
+			return news[400];
+		case ArticlePillar.Culture:
+			return culture[400];
+		case ArticlePillar.Lifestyle:
+			return lifestyle[400];
+		case ArticlePillar.Sport:
+			return sport[400];
+		case ArticlePillar.Opinion:
+			return opinion[400];
+		case ArticleSpecial.Labs:
+			return labs[400];
+		case ArticleSpecial.SpecialReport:
+			return specialReport[400];
+	}
+}
+
+const textInverted = (format: ArticleFormat): Colour => {
+	switch(format.theme) {
+		case ArticlePillar.News:
+			return news[500];
+		case ArticlePillar.Culture:
+			return culture[500];
+		case ArticlePillar.Lifestyle:
+			return lifestyle[500];
+		case ArticlePillar.Sport:
+			return sport[500];
+		case ArticlePillar.Opinion:
+			return opinion[500];
+		case ArticleSpecial.Labs:
+			return specialReport[500];
+		case ArticleSpecial.SpecialReport:
+			return specialReport[500];
+	}
+}
```

Comment:
Will people know what `kicker` and `inverted` mean? Are they holdover terms from the old way of doing things? Can the switch statements in `textKicker` and `textInverted` be inlined into the `textStandfirstLink` function instead?

I think one reason for using `decidePalette` and `editorialPalette` is that it's very clear what a colour is and where it's used. That way it's easier for people, Design included, to come in and tweak a colour in isolation, without worrying about the change affecting multiple things.

Besides, I think there's already some discrepancy: `textStandfirstLink` says labs pieces should use `labs[300]` but `textKicker` says it should be `labs[400]` 🤔.